responses/fr/HassRespond.yaml

```diff
@@ -0,0 +1,6 @@
+language: fr
+responses:
+  intents:
+    HassRespond:
+      hello: "Bonjour depuis Home Assistant."
```

Comment:
Looks weird to say that but ok 😅

Reply:
Do you have a better idea? I am 100% ok to change it if you find better!